lib/manageiq/api/common/application_controller_mixins/api_doc.rb

```diff
@@ -28,8 +33,8 @@ def api_doc_definition
               @api_doc_definition ||= api_doc.definitions[model.name]
             end
 
-            def api_version
-              @api_version ||= name.split("::")[1].downcase
+            def api_version(version = nil)
```

Comment:
This would allow caching the incorrect version on a class.  Do we want that?

Reply:
agreed, I'm preferring this approach now: lib/manageiq/api/common/application_controller_mixins/request_body_validation.rb